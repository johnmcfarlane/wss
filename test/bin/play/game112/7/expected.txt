 54  1,6  | JHOW
 33  4,8  - QUI
 28  3,7  - HOWS
 26  2,7  - WHO
 25  7,3  | QUI
 24  5,5  - WAQF
 23  7,2  | FIQH
 22  3,7  - FI
 22  3,7  - WO
 21  4,4  | FUGIO
 20  4,6  | GOWF
 20  1,7  - OW
 19  2,7  - IF
 19  5,3  | WIFE
 18  3,7  - HI
 18  3,7  - HO
 18  1,7  - OH
 18  3,3  | WIFE
 18  2,5  | WaQF
 17  3,5  | FEW
 17  4,4  | FIGO
 17  4,4  | FUG
 17  1,4  | FUJI
 17  4,3  | WHIG
 17  8,4  - WHOO
 16  4,4  | FIG
 16  4,4  | FOG
 16  7,3  | FOH
 16  4,6  | GIF
 16  2,7  - HOI
 16  1,4  | HOJU
 16  7,3  | HOW
 16  3,3  | HOWE
 16  2,4  | QUaW
 16  5,5  - WAUF
 16  4,4  | WIG
 15  6,5  - AQUO
 15  3,5  | FEH
 15  5,5  | FEU
 15  3,5  | HEW
 15  4,4  | HUGO
 15  4,7  - IFS
 15  1,6  | JOW
 15  3,4  | QUE
 15  5,5  - WAIF
 15  4,3  | WOUGH
 14  4,4  | HUG
 14  1,7  - OI
 14  5,5  - WAF
 13 11,9  | DUI
 13 11,9  | DUO
 13  5,5  - FAH
 13  3,5  | FEU
 13  4,4  | HOG
 13  4,7  - IHS
 13  4,7  - OHS
 13  7,4  | QI
 13  2,3  | QUIa
 13  9,4  - QUO
 13  4,10 - QUO
 13  2,4  | QUaI
 13  1,5  | UJI
 13  5,5  - WAH
 13  2,7  - WO
 13  2,6  | aQUO
 12  5,5  - HAF
 12  5,5  - HAW
 12  4,8  - HUI
 12  8,3  - HUIC
 12  2,4  | QUa
 11  5,5  | HEI
 11  9,3  - HIC
 11  2,5  - HO
 11  2,7  - HO
 11  9,3  - HOC
 11  4,8  - HOI
 11  7,3  | HUI
 11  5,4  | IFE
 11  1,6  | JO
 11  4,4  | OUGH
 11  5,8  - QI
 11  3,5  | WEI
 10  5,5  - FA
 10  5,5  | FE
 10  4,6  | GOU
 10  5,5  - HAU
 10  7,3  | HOI
 10  3,4  | IFE
 10  3,7  - OI
 10  7,4  | OW
 10  3,4  | OWE
 10  5,10 | WO
 10  2,5  | WaUF
  9  3,6  | EF
  9  3,6  | EW
  9  3,5  | FE
  9  4,6  | GHI
  9  5,5  - HAO
  9  3,5  | HEI
  9  5,7  - HSI
  9  7,4  | UH
  9  3,5  | WE
  9  3,10 - WHOO
  9  2,4  | WHaU
  9  2,5  | WaIF
  8  5,5  - HA
  8  5,5  | HE
  8  5,10 | HI
  8  5,10 | HO
  8  3,4  | HUE
  8  4,8  - IWI
  8  5,5  - OAF
  8  7,4  | OH
  8  8,4  - OHIO
  8  4,5  | UGH
  8  9,4  - WHO
  8  4,10 - WHO
  8  2,3  | WHOa
  8  2,5  | WaF
  7  6,5  - AUF
  7  3,6  | EH
  7  7,4  | FI
  7  3,4  | FIE
  7  3,4  | FOE
  7  2,5  | FaH
  7  4,6  | GIO
  7  4,6  | GOI
  7  3,5  | HE
  7  2,5  | HaF
  7  2,5  | HaW
  7  5,10 | IF
  7  2,7  - IO
  7  5,10 | OF
  7  7,3  | OUI
  7  5,10 | OW
  7  9,4  - UFO
  7  4,10 - UFO
  7  2,4  | WHa
  7  3,4  | WOE
  7  2,5  | WaH
  6  6,5  - AHU
  6  6,5  - AUH
  6  9,4  - FOO
  6  4,10 - FOO
  6  7,4  | HI
  6  3,4  | HIE
  6  3,4  | HOE
  6  2,3  | HUIa
  6  9,4  - OFO
  6  4,10 - OFO
  6  5,10 | OH
  6  3,10 - OHIO
  6  4,8  - OUI
  6  9,4  - WOO
  6  4,10 - WOO
  6  2,6  | aUF
  5  6,5  - AHI
  5  6,5  - AHO
  5  6,5  - AW
  5  5,8  - FI
  5  9,4  - HOO
  5  4,10 - HOO
  5  2,5  | HaU
  5  2,4  | IWa
  5  2,3  | OHIa
  5  9,4  - OHO
  5  4,10 - OHO
  5  2,5  | OaF
  5 10,4  - WO
  5  5,10 - WO
  5  2,6  | aHU
  5  2,6  | aUH
  4  6,5  - AH
  4  5,6  | EO
  4  2,5  | Fa
  4  4,6  | GI
  4  4,6  | GO
  4  5,8  - HI
  4  2,4  | HIa
  4 10,4  - HO
  4  5,10 - HO
  4  2,5  | HaO
  4  5,10 | IO
  4  7,4  | OI
  4  5,10 | OI
  4  5,7  - OS
  4  6,7  - SH
  4  2,6  | aHI
  4  2,6  | aHO
  4  2,6  | aW
  3  3,6  | EO
  3  2,5  | Ha
  3  3,5  | OE
  3  2,6  | aH
  2  6,5  - AI
  2 10,4  - IO
  2  5,10 - IO
  2  2,5  | IaO
  2  5,8  - OI
  2 10,4  - OO
  2  5,10 - OO
  2  6,7  - SI
  2  6,7  - SO
  1  2,6  | aI
