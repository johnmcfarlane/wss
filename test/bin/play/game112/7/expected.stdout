 33  4,8  - QUI
 28  3,7  - HOWS
 26  2,7  - WHO
 25  7,3  | QUI
 24  5,5  - WAQF
 23  7,2  | FIQH
 22  3,7  - FI
 22  3,7  - WO
 21  4,4  | FUGIO
 20  1,7  - OW
 19  2,7  - IF
 19  5,3  | WIFE
 18  3,7  - HI
 18  3,7  - HO
 18  1,7  - OH
 18  3,3  | WIFE
 18  2,5  | WaQF
 17  3,5  | FEW
 17  4,4  | FUG
 17  1,4  | FUJI
 17  4,3  | WHIG
 16  4,4  | FIG
 16  4,4  | FOG
 16  7,3  | FOH
 16  2,7  - HOI
 16  7,3  | HOW
 16  3,3  | HOWE
 16  4,4  | WIG
 16  4,4  | WOG
 15  3,5  | FEH
 15  5,5  | FEU
 15  3,5  | HEW
 15  4,7  - IFS
 15  1,6  | JOW
 15  3,4  | QUE
 15  5,5  - WAIF
 14  4,4  | HUG
 14  1,7  - OI
 13  5,5  - FAH
 13  3,5  | FEU
 13  4,4  | HOG
 13  4,7  - OHS
 13  7,4  | QI
 13  2,4  | QUaI
 13  2,7  - WO
 12  5,5  - HAW
 12  8,3  - HUIC
 12  2,4  | QUa
 11  9,3  - HIC
 11  2,5  - HO
 11  2,7  - HO
 11  4,8  - HOI
 11  1,6  | JO
 11  5,8  - QI
 10  5,5  - FA
 10  5,5  | FE
 10  7,3  | HOI
 10  3,7  - OI
 10  7,4  | OW
 10  3,4  | OWE
 10  5,10 | WO
  9  3,6  | EF
  9  3,6  | EW
  9  3,5  | FE
  9  4,6  | GHI
  9  5,5  - HAO
  9  7,4  | UH
  9  3,5  | WE
  9  2,5  | WaIF
  8  5,5  - HA
  8  5,5  | HE
  8  5,10 | HI
  8  5,10 | HO
  8  3,4  | HUE
  8  4,8  - IWI
  8  5,5  - OAF
  8  7,4  | OH
  8  4,5  | UGH
  8  9,4  - WHO
  8  4,10 - WHO
  8  2,3  | WHOa
  7  3,6  | EH
  7  7,4  | FI
  7  3,4  | FIE
  7  3,4  | FOE
  7  2,5  | FaH
  7  3,5  | HE
  7  2,5  | HaW
  7  5,10 | OF
  7  5,10 | OW
  7  2,4  | WHa
  7  3,4  | WOE
  6  7,4  | HI
  6  3,4  | HIE
  6  3,4  | HOE
  6  5,10 | OH
  6  9,4  - WOO
  6  4,10 - WOO
  5  6,5  - AHI
  5  6,5  - AW
  5  5,8  - FI
  5  2,3  | OHIa
  5  9,4  - OHO
  5  4,10 - OHO
  5  2,5  | OaF
  5 10,4  - WO
  5  5,10 - WO
  4  6,5  - AH
  4  5,6  | EO
  4  2,5  | Fa
  4  4,6  | GI
  4  4,6  | GO
  4  5,8  - HI
  4 10,4  - HO
  4  5,10 - HO
  4  2,5  | HaO
  4  7,4  | OI
  4  5,10 | OI
  4  5,7  - OS
  4  6,7  - SH
  4  2,6  | aHI
  4  2,6  | aW
  3  3,6  | EO
  3  2,5  | Ha
  3  3,5  | OE
  3  2,6  | aH
  2  6,5  - AI
  2  5,8  - OI
  2 10,4  - OO
  2  5,10 - OO
  2  6,7  - SI
  2  6,7  - SO
  1  2,6  | aI
